### ISSUE TITLE
[Question] how could I use llava-v1.6 in command line?

### ISSUE THREAD

**Issue by erjiaxiao** [haotian-liu/LLaVA#1259]
### Question

Hello! Could I directly replace the model in the following command to use llava-v1.6?

`python -m llava.serve.cli --model-path liuhaotian/llava-v1.6-34b --image-file "https://llava-vl.github.io/static/images/view.jpg" --load-4bit`

Thank you!

**Comment by phuchm** [comment]
Of course, you can do it, but remember that use liuhaotian/llava-v1.6-34b will take so much resource! Please refer to https://github.com/haotian-liu/LLaVA/issues/1099 for more detail!

**Comment by erjiaxiao** [comment]
Thanks a lot! @phuchm
